File: app/src/main/java/com/example/myapplication/models/mangamodels/MangaNewReleaseResultModel.java
package com.example.myapplication.models.mangamodels;

import androidx.annotation.NonNull;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

import lombok.Data;

@Data
public class MangaNewReleaseResultModel implements Serializable {

    private String mangaType;
    private String mangaTitle;
    private String mangaThumb;
    private String mangaDetailURL;
    private List<LatestMangaDetailModel> latestMangaDetail = new ArrayList<>();


    @NonNull
    @Override
    public String toString() {
        return "MangaNewReleaseResult{" +
                "mangaType='" + mangaType + '\'' +
                ", mangaTitle='" + mangaTitle + '\'' +
                ", mangaThumb='" + mangaThumb + '\'' +
                ", mangaDetailURL='" + mangaDetailURL + '\'' +
                ", latestMangaDetail=" + latestMangaDetail +
                '}';
    }

    @Data
    public class LatestMangaDetailModel implements Serializable {
        private List<String> chapterTitle;
        private List<String> chapterURL;
        private List<String> chapterReleaseTime;

        @NonNull
        @Override
        public String toString() {
            return "LatestMangaDetailModel{" +
                    "chapterTitle=" + chapterTitle +
                    ", chapterURL=" + chapterURL +
                    ", chapterReleaseTime=" + chapterReleaseTime +
                    '}';
        }
    }
}

File: app/src/main/java/com/example/myapplication/models/mangamodels/MangaNewReleaseResultModel.kt
package com.example.myapplication.models.mangamodels

import java.io.Serializable
import java.util.ArrayList

data class MangaNewReleaseResultModel(
        var mangaType: String? = null,
        var mangaTitle: String? = null,
        var mangaThumb: String? = null,
        var mangaDetailURL: String? = null,
        var isMangaStatus: Boolean = false,
        var latestMangaDetail: List<LatestMangaDetailModel> = ArrayList()
) : Serializable {

    data class LatestMangaDetailModel(
            var chapterTitle: List<String>? = null,
            var chapterURL: List<String>? = null,
            var chapterReleaseTime: List<String>? = null
    ) : Serializable {

        override fun toString(): String {
            return "LatestMangaDetailModel{" +
                    "chapterTitle=" + this.chapterTitle +
                    ", chapterURL=" + this.chapterURL +
                    ", chapterReleaseTime=" + this.chapterReleaseTime +
                    '}'.toString()
        }
    }

    override fun toString(): String {
        return "MangaNewReleaseResult{" +
                "mangaType='" + this.mangaType + '\''.toString() +
                ", mangaTitle='" + this.mangaTitle + '\''.toString() +
                ", mangaThumb='" + this.mangaThumb + '\''.toString() +
                ", mangaDetailURL='" + this.mangaDetailURL + '\''.toString() +
                ", latestMangaDetail=" + this.latestMangaDetail +
                '}'.toString()
    }
}
